app/scripts/modules/cloudfoundry/serverGroup/configure/serverGroupArtifactSettingsDirective.html

```diff
@@ -0,0 +1,54 @@
+<ng-form name="artifactSettings">
+  <div class="form-group">
+    <div class="col-md-3 sm-label-left">
+      <b>Repository</b>
+    </div>
+    <div class="col-md-7">
+      <input type="text"
+             class="form-control input-sm"
+             name="repository"
+             ng-model="command.repository"/>
+    </div>
+  </div>
+  <div class="form-group">
+    <div class="col-md-3 sm-label-left">
+      <b>Artifact</b>
+    </div>
+    <div class="col-md-7">
+      <input type="text"
+             class="form-control input-sm"
+             name="artifact"
+             ng-model="command.artifact"/>
+    </div>
+  </div>
+  <div class="form-group">
+    <div class="col-md-3 sm-label-left">
+      <b ng-if="!artifactSettingsCtrl.isS3()">Username</b>
+      <b ng-if="artifactSettingsCtrl.isS3()">Access Key</b>
+    </div>
+    <div class="col-md-7">
+      <input type="text"
+             class="form-control input-sm"
+             name="username"
+             ng-if="!artifactSettingsCtrl.isS3()"
+             ng-model="command.username"/>
+      <input type="password"
+             class="form-control input-sm"
+             name="username"
+             ng-if="artifactSettingsCtrl.isS3()"
+             ng-model="command.username"/>
+    </div>
+  </div>
+  <div class="form-group">
+    <div class="col-md-3 sm-label-left">
+      <b ng-if="!artifactSettingsCtrl.isS3()">Password</b>
+      <b ng-if="artifactSettingsCtrl.isS3()">Secret Key</b>
+    </div>
+    <div class="col-md-7">
+      <input type="password"
+             class="form-control input-sm"
+             name="password"
+             ng-model="command.password"/>
+    </div>
+  </div>
```

Comment:
@anotherchrisberry Can I override the label's class value? I noticed that "Access Key" and "Secret Key" were a bit too long for col-md-2, so mine used col-md-3.